blog/2023-03-10-conda-survey.md

```diff
@@ -0,0 +1,194 @@
+
+[![Data formats, top](/img/2023-03-10-conda-survey/data-formats-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1927071597)
+
+Over 33 data formats were listed as being used. CSV and JSON were the most popular with 6 out of every 7 responders using CSV.
+
+[![Databases, top](/img/2023-03-10-conda-survey/databases-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1314196509)
+
+Respondents listed 13 database management systems they use.  SQLite (36%) and PostgreSQL (32%) led this group, followed by MySQL (19%).
+
+## Tools
+
+[![Statistical analysis tools, top](/img/2023-03-10-conda-survey/statistical-analysis-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1524339820)
+
+Microsoft Excel (35%) and R (31%) were the two most popular tools in the statistical analysis tools category.
+
+[![ETL Tools, top](/img/2023-03-10-conda-survey/etl-tools-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1574024089)
+[![Data visualization tools, top](/img/2023-03-10-conda-survey/data-visualization-tools-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1532616702)
+
+The survey asked specific questions about ETL and data visualization tool use.  20 ETL tools had users, but only Pandas (54%), Apache Airflow (10%) and Apache Spark (10%) had more than one or two.  Jupyter (67%) was the clear leader in data visualization responses, with Plotly (35%) following.  23 other visualization tools were listed, most with only one or two users.
+
+
+# Who uses conda?
+
+71 of 72 responders write some code as part of their work.
+
+## Participant experience
+
+Both the conda and Python groups had similar levels of experience. [68% of conda respondents had 4+ years of experience with conda](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1883612863), while 66% of Python respondents has 4+ years of experience with Python.  The conda survey asked participants about their job functions, industries and Project types.
+
+## Job Functions
+
+[![Job functions](/img/2023-03-10-conda-survey/job-function-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1839470285)
+
+The most common role was *Developer*, followed closely by *Research Scientists.* The top 12 roles can be roughly grouped:
+
+* *Developer* (29.2%)
+* *Research Scientist* (27.8%) and *Applied Scientist* (15.3%)
+* *Data Scientist* (22.2%), *Data Engineer* (15.3%), and *ML Scientist* (12.5%)
+* *Professor/Instructor* (12.5%) and *Student* (11.1%)
+* *ML Ops* (6.9%), *DevOps* (5.6%), and *System Administrator* (4.2%)
+* *Project Manager* (4.2%)
+
+People could select more than one job function.
```

Comment:
Oh, it ended already :(

I was expecting something like a conclusion or a closing statement or something like that. The end was a bit abrupt and could use a goodbye :D

Reply:
Moved closing from the bottom of the top to the bottom of the bottom, and expanded it slightly